src/index.html

```diff
@@ -7,6 +7,43 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar month-length start-day">
+      <div class="calendar__day calendar__day-sun"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
you don't need those classes, you should use modifiers to set the start day and the month length